components/brave_wallet_ui/common/async/lib.ts

```diff
@@ -190,8 +191,9 @@ export function refreshBalances () {
     const { jsonRpcService } = getAPIProxy()
     const { wallet: { accounts, userVisibleTokensInfo } } = getState()
 
+    // TODO Pass chainId parameter
     const getBalanceReturnInfos = await Promise.all(accounts.map(async (account) => {
-      const balanceInfo = await jsonRpcService.getBalance(account.address, account.coin)
+      const balanceInfo = await jsonRpcService.getBalance(account.address, account.coin, "<chainId>")
```

Comment:
`chainId` is not a property of account, so we will need to get that value elsewhere.  I think we could loop through the `networkList` to produce a map from `coin => [chainId]`, and when requesting balances, loop through each `chainId` for the `account.coin`.

But maybe it makes sense to change the data model slightly to avoid this preprocessing if this is something we will do frequently.

Reply:
Currently I am working on a refactor for the UI that will do this looping/mapping.
For the moment you could pass `selectedNetwork.chainId`,
you can get the `selectedNetwork` state like so
```const { wallet: { accounts, userVisibleTokensInfo, selectedNetwork } } = getState()```
from this line https://github.com/brave/brave-core/pull/12530/files#diff-b85091d7cf18c04a6a4ddd6a277db8bbe711a12299e16e98c0a3e7e11f96c00eR192